src/v2/Apps/Auction/auctionRoutes.tsx

```diff
@@ -48,6 +49,16 @@ const AuctionFAQRoute = loadable(
 )
 
 export const auctionRoutes: AppRouteConfig[] = [
+  {
+    // Redirect from the old route to the new one
+    path: "/sale/:slug?",
+    children: [
+      new Redirect({
+        from: "/",
+        to: "/auction/:slug?",
+      }) as any,
```

Comment:
https://4catalyzer.github.io/found/advanced/redirects

Shouldn't this be `from: "/sale/:slug?",`

Reply:
I tried this yesterday before Jenna gave us the go ahead to just remove /sale and it didn't quite work. I bet this does tho...

Reply:
Based on how I'm reading it, no, but maybe! I haven't worked with the router before, but glad to check that and change it if it works.

Reply:
I'm confused why would be redirecting from the root.

Reply:
I saw this method of redirecting in orderRoutes and consignRoutes:

https://github.com/artsy/force/blob/9e9af2114ff29b0565e10b88a80e9e081afec954/src/v2/Apps/Order/orderRoutes.tsx#L305-L309

It seems a little odd to me, but it was the only way I found to achieve this with my limited knowledge of Force internals.

I don't think it's redirecting from the root, but more like the root of the path that it's nested under, but I might be wrong!

Reply:
So does specifying it as `sale/:slug` simply not work? What about `*`?

Reply:
But sure, I get what you're saying.